src/config.rs

```diff
@@ -149,6 +150,8 @@ pub(crate) struct Connector {
 
     //pub(crate) on_pause: PauseBehaviour,
     pub(crate) metrics_interval_s: Option<u64>,
+
+    pub(crate) initial_command: Option<Command>,
```

Comment:
could we make this `commands: Vec<Command>` instead of an option. This would help in, for example a UDP socket where you want to open and set a few flags, or a file connector that wants to open multiple files etc.